README.md

```diff
@@ -56,18 +56,19 @@ npm install @genezio/types
 yarn add @genezio/types
 ```
 # Documentation
-To find more details on how to use `@genezio/types`, check out the official [documentation](https://genez.io/docs/types)
+To find more details on how to use `@genezio/types`, check out the official [documentation](https://genezio.com/docs/features/backend-deployment/)
 
 Here you can find some examples:
 - [Webhook example](https://github.com/Genez-io/genezio-examples/tree/master/typescript/webhook)
+- [Auth example](https://github.com/Genez-io/genezio-react-auth)
```

Comment:
We now have two examples for auth and my feeling says that we will keep adding and updating tutorials in the future.

I'd say to change this call to action to the redirect to the tutorials page on our documentation:
https://genezio.com/docs/tutorials